src/plugins/discourse/mirror.test.js

```diff
@@ -154,11 +200,42 @@ class MockFetcher implements Discourse {
       topicId: topicId,
       authorUsername: NullUtil.orElse(username, "credbot"),
       cooked: NullUtil.orElse(cooked, "<h1>Hello World</h1>"),
+      timestampMs: 2000 + postId,
     };
     this._posts.set(postId, postInfo);
     return postId;
   }
 
+  editPost(postId: PostId, changes: $Shape<PostInfo>): void {
```

Comment:
Not asking for a change at this point, but would have preferred to have all these updates to test code in a separate commit. (Add the new test methods, then use them to test the new update mode.)